.travis.xacro_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
whatever I try, it does not find `rosrun`....even adding `source /opt/ros/$ROS_DISTRO/setup.bash` before the for-loop does not help...

```
ici_fold:start:after_script
ici_time:start:v3pkai10>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
XACRO TEST
Testing /root//home/fxm/projects/indigo/care-o-bot/src/cob_calibration_data/raw3-1/calibration/calibration_offset.urdf.xacro
./.travis.xacro_test.sh: line 7: rosrun: command not found
```